app/views/pages/home.html.erb

```diff
@@ -59,3 +59,18 @@
         </div>
       </div>
     </div>
+
+
+<% content_for(:after_js) do %>
+ <script>
```

Comment:
It probably didn't work because the syntax of your JS is broken (see other comment, you need to remove the `<script></script>` tags from the `doorbell.js` file).